### PR TITLE
feat(Card): Add background and border styles

### PR DESCRIPTION
**Pull request by pompomon**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Add default background and border styles for the Card component

image
image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12790)

### CODE REVIEW

File: packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts [hunk 1]
```diff
@@ -39,25 +42,32 @@ const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> =
...
       }),
 
       ...(p.disabled && {
         cursor: 'not-allowed',
         color: v.colorDisabled,
+        backgroundColor: v.backgroundColorDisabled,
         borderColor: v.borderColorDisabled,
         boxShadow: v.boxShadowDisabled,
         ':hover': {
+          backgroundColor: v.backgroundColorDisabled,
```
Comment (silviuaavram) [inline packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts:62]: why were these 3 needed? is there an override between them and the one on line 58?

Comment (pompomon) [reply packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts:62]: Nope, good point, will remove them :)

File: packages/fluentui/CHANGELOG.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `onClick` prop and focusable behaviors to `Card` component @pompomon ([#12688](https://github.com/microsoft/fluentui/pull/12688))
 - Add `disabled` prop and styles to `Card` component @pompomon ([#12743](https://github.com/microsoft/fluentui/pull/12743))
 - Adding multiselection for the `Tree` @kolaps33, @assuncaocharles ([#12362](https://github.com/microsoft/fluentui/pull/12362))
+- Add `background` and `border` colors to `Card` @pompomon ([#12790](https://github.com/microsoft/fluentui/pull/12790))
```
Comment (mnajdova) [inline packages/fluentui/CHANGELOG.md:52]:
```suggestion
- Add `background` and `border` colors to `Card` in Teams theme @pompomon ([#12790](https://github.com/microsoft/fluentui/pull/12790))
```

File: packages/fluentui/react-northstar/src/themes/teams/components/Card/cardVariables.ts [hunk 3]
```diff
@@ -39,11 +46,20 @@ export interface CardVariables {
...
   return {
+    backgroundColor: siteVars.colorScheme.default.background,
+    backgroundColorDisabled: siteVars.colorScheme.default.backgroundDisabled,
+    backgroundColorFocus: siteVars.colorScheme.default.backgroundFocus,
+    backgroundColorHover: siteVars.colorScheme.default.backgroundHover,
+    backgroundColorPressed: siteVars.colorScheme.default.backgroundPressed,
     borderStyle: 'solid',
     borderColor: siteVars.colors.black,
+    borderColorDisabled: siteVars.colorScheme.default.borderDisabled,
+    borderColorHover: siteVars.colors.black,
```
Comment (mnajdova) [inline packages/fluentui/react-northstar/src/themes/teams/components/Card/cardVariables.ts:57]: This is not safe for theme switching, can you check with design, or if it's not by redline, let's add some color scheme value that will look decent in all themes.

Comment (pompomon) [reply packages/fluentui/react-northstar/src/themes/teams/components/Card/cardVariables.ts:57]: I'n still waiting for redlines :)
OK, I'll switch to some other color defined everywhere

**Pull request merged**